firefox-ios/Client/Frontend/Settings/RemoteSettingsOptionViewController.swift

```diff
@@ -0,0 +1,141 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+class RemoteSettingsOptionViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: view.bounds)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(cellType: OneLineTableViewCell.self)
+        return tableView
+    }()
+    private var collections: [ServerCollection] = []
+    private var remoteSettingsUtil = RemoteSettingsUtil(bucket: .defaultBucket, collection: .searchTelemetry)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupNavigationBar()
+        setupTableView()
+        fetchCollections(bucketID: Remotebucket.defaultBucket.rawValue)
+    }
+
+    func setupNavigationBar() {
+        let settingsButton = UIBarButtonItem(image: UIImage(systemName: "gearshape"),
```

Comment:
Yup, can also be ignored because final version won't have this gear but this is fine for exploring in this PR